src/SolidityLauncher.ts

```diff
@@ -496,23 +460,31 @@ export class SolidityLauncher {
       const sampler = new SolidityRandomSampler(currentSubject);
 
       const crossover = new SolidityTreeCrossover();
-      const algorithm = createAlgorithmFromConfig(sampler, runner, crossover);
+      const algorithm = createSearchAlgorithmFromConfig(
+        this.pluginManager,
+        null,
+        sampler, 
+        runner, 
+        crossover
+        );
 
-      await suiteBuilder.clearDirectory(Properties.temp_test_directory);
+      await suiteBuilder.clearDirectory(CONFIG.tempTestDirectory);
 
       // allocate budget manager
-      const iterationBudget = new IterationBudget(Properties.iteration_budget);
+      const iterationBudget = new IterationBudget(CONFIG.iterationBudget);
       const evaluationBudget = new EvaluationBudget();
```

Comment:
We don't have a config for the evaluation budget?

Reply:
We do, but this was how i found it. Added the config now.

Reply:
I don't see the fix in the PR

Reply:
Weird I dont see it either in the PR but if you look in the file on the branch it is there.